reference.md

```diff
@@ -82,14 +82,6 @@ filter
     they read data from [standard input](#standard-input),
     process it, and write the result to [standard output](#standard-output).
 
-flag
-:   A terse way to specify an option or setting to a command-line program.
```

Comment:
This is mostly repeated from the option entry, but without a definition of the difference. I would probably drop this entry, and maybe mention in the option entry this alternative terminology.

Reply:
This entry was renamed to 'option' and slightly cleaned of subjective ("terse way") wording.

Reply:
Sorry, I was reading this with tired eyes and got confused, thinking there were not yet entries.